sounds/views.py

```diff
@@ -495,11 +495,19 @@ def pack_edit(request, username, pack_id):
             return HttpResponseRedirect(pack.get_absolute_url())
     else:
         form = PackEditForm(instance=pack, initial=dict(pack_sounds=pack_sounds))
-        current_sounds = Sound.objects.bulk_sounds_for_pack(pack_id=pack.id)
+        current_sounds = list(Sound.objects.bulk_sounds_for_pack(pack_id=pack.id))
```

Comment:
Why do we do the `list()` here? Is this because otherwise when we do `paginate` it throws some errors? Django's paginator is supposed to receive a queryset and then runs the query adding a range at the end. In our case we probably can't do that because `Sound.objects.bulk_sounds_for_pack()` does not accept a range at the end (am I right?). Anyway, if we turn this into a list, then the query is executed and we get the info for all sounds, not only for those needed for the `n-th` page.

The point is the paginator is to improve the speed of the page and also the UI. I think this solution only improves the UI but not the speed of the page. I'm not sure however how should be go about it.

Reply:
yes, you're right. I changed the bulk query for a regular django query